chains/oev-network-sepolia-testnet.json

```diff
@@ -0,0 +1,24 @@
+{
+  "alias": "oev-network-sepolia-testnet",
+  "blockTimeMs": 1686780,
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://oev-network-sepolia-testnet-blockscout.eu-north-2.gateway.fm/api/"
```

Comment:
I was holding off to figure this one out, apparently it doesn't work without the trailing `/`